ambry-store/src/test/java/com.github.ambry.store/CuratedLogIndexState.java

```diff
@@ -512,7 +574,7 @@ private void setupTestState(boolean isLogSegmented, long segmentCapacity, StoreM
     }
     // make sure all indexes are written to disk and mapped as required (forcing IndexPersistor to run).
     log.flush();
-    reloadIndex(true, false);
+    //reloadIndex(true, false);
```

Comment:
This is commented out because I don't have the full changes to IndexSegment for serializing and deserializing. Reloading the index will cause mismatch of lastModifiedTime with the state's `time`. Once #541 is merged this can be uncommented.